src/ws/login.html

```diff
@@ -25,8 +25,9 @@
         <div id="login" class="login-area" style="visibility: hidden;">
           <div role="form">
 
-            <div id="error-group" class="alert alert-danger" hidden>
-              <span id="login-error-message"></span>
+            <div id="error-group" class="pf-c-alert pf-m-danger pf-m-inline dialog-error group-hidden" aria-label="inline danger alert">
+              <svg style="vertical-align:-.125em" fill="currentColor" viewBox="0 0 512 512" aria-hidden="true"><path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/></svg>
+              <h4 id="login-error-message" translatable="yes" class="pf-c-alert__title"></h4>
```

Comment:
The `translatable` doesn't seem to make sense here. There's no static string to parse out here other than the empty string, which may or may not confuse our parser?

Reply:
In addition we stopped using translatable in favor of the valid translate attr - see https://github.com/cockpit-project/cockpit/pull/13133